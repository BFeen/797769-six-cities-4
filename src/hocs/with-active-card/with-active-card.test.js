import React from "react";
import renderer from "react-test-renderer";
import withActiveCard from "./with-active-card.js";


const MockComponent = () => {
  return (
    <div></div>
  );
};

MockComponent.propTypes = {};

const MockComponentWrapped = withActiveCard(MockComponent);

describe(`WithActiveCard snapshot testing`, () => {
  it(`HOC WithActiveCard rendering correctly`, () => {
    const tree = renderer.create(
        <MockComponentWrapped
          onCardMouseEnter={() => {}}
          onCardMouseLeave={() => {}}
          activeCard={{}}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
