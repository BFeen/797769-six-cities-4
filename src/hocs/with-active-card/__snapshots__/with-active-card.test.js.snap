// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WithActiveCard snapshot testing HOC WithActiveCard rendering correctly 1`] = `<div />`;
